.circleci/config.yml

```diff
@@ -0,0 +1,91 @@
+version: 2
+jobs:
+  build:
+
+    # Primary command image where all commands run
+    docker:
+      - image: circleci/ruby:2.4.2-node-browsers
+        environment:
+          RAILS_ENV: test
+          ORANGELIGHT_HOST: localhost
+          ORANGELIGHT_USER: postgres
+
+      # Service container image available at 'host: localhost'
+      - image: circleci/postgres:9.6-alpine    
+        environment:
+          POSTGRES_USER: orangelight
+          POSTGRES_DB: orangelight_test
+          POSTGRES_PASSWORD: ""
+
+    working_directory: ~/orangelight
+
+    steps:
+      - checkout
+      #set timezone to est
+      - run:
+          name: Set Timezone to EST
+          command: echo 'America/New_York' = /etc/timezone
+      
+      # Restore bundle cache
+      - restore_cache:
+          keys:
+          - orangelight-{{ checksum "Gemfile.lock" }}
+          - orangelight-
+      
+      # Bundle install dependencies
+      - run:
+          name: Install dependencies
+          command: bundle check --path=vendor/bundle || bundle install --path=vendor/bundle --jobs 4 --retry 3
+      
+      - run: sudo apt install postgresql-client
```

Comment:
Do you actually need to do this?

Reply:
The seed file includes a psql command. Before adding the postgres-client it wasn't able to see psql and run the seed and set up the database.